spec/index.bs

```diff
@@ -1459,6 +1459,13 @@ GPUTexture includes GPUObjectBase;
     ::
         The allowed usages for this {{GPUTexture}}.
 
+    : <dfn>\[[viewFormats]]</dfn> of type `sequence<{{GPUTextureFormat}}>`.
+    ::
+        The additional formats that are allowed to be used as {{GPUTextureViewDescriptor}}.{{GPUTextureViewDescriptor/format}}
+        for views created from this {{GPUTexture}}.
+
+        The base texture format {{GPUTextureFormatDescriptor/format}} is always allowed.
+
```

Comment:
The spec text you're adding doesn't match the title of the PR.  The PR title talks about `GPUTextureDescriptor.viewformats` but here you're adding the sequence of formats to the `GPUTexture` object itself.

For D3D12, you need to tell the API that you want a typeless texture at object creation time.  So specifying something like `viewFormats` would be best as part of the descriptor as the title of the PR states.  

The best place to look at the mapping for compatible formats for D3D can be found in the format tables for the various feature levels.  https://microsoft.github.io/DirectX-Specs/d3d/archive/images/d3d11/D3D11_3_Formats_FL11_1.xls is a link to the one for FL11_1.

Reply:
The section you commented on contains the "internal slots" for the texture. It saves creation arguments so they can be later used by other parts of the spec for validation. The real API change is [1] which adds `viewFormats` to the `GPUTextureDescriptor`.

Thanks for the link, [lease help build the compatibility classes for WebGPU in #744

Reply:
Thank you, Corentin.  I admit I overlooked the viewFormats section.  

I think we should go forward with this PR and amend the spec pending the outcome of #774 about which ones are allowed to go together.